editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -201,12 +205,19 @@ impl Fsm for PathToolFsmState {
 							if toggle_add_to_selection {
 								responses.push_back(DocumentMessage::AddSelectedLayers { additional_layers: intersection }.into());
 							} else {
+								// Selects the topmost layer when selecting intersecting shapes
```

Comment:
This doesn't seem to be true. When I test it with two intersecting shapes, both end up being selected.

Question for @TrueDoctor: is it even necessary to pick the top one, or would the occlusion normally be taken into account? (In other words, for selecting shapes with the Select tool, does that have to be manually filtered for what's on top like the code here?)

Reply:
> This doesn't seem to be true. When I test it with two intersecting shapes, both end up being selected. Please fix that so only the top one is selected.
This is probably caused by the SetSelectedLayers message on line 210 I would guess

As for your question, I'm not sure about that would have to look that up